packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -271,7 +271,7 @@ class SingleChildScrollView extends StatelessWidget {
         child: scrollable,
         onNotification: (ScrollUpdateNotification notification) {
           final FocusScopeNode currentScope = FocusScope.of(context);
-          if (notification.dragDetails != null && !currentScope.hasPrimaryFocus && currentScope.hasFocus) {
+            if (!currentScope.hasPrimaryFocus && currentScope.hasFocus) {
```

Comment:
The indentation is still wrong here.

Reply:
if i am removing space available between `if (` then linux analyze is failing here, though on my local system on running `flutter analyze` its showing no issues

Reply:
Try this?

```suggestion
          if (!currentScope.hasPrimaryFocus && currentScope.hasFocus) {
```